src/web/app/pages-session/session-submission-page/saving-complete-modal/saving-complete-modal.component.html

```diff
@@ -35,15 +35,16 @@ <h5 class="modal-title">
     Note that you can change your responses and submit them again any time before the session closes.
   </p>
   <p *ngIf="!hasFailToSaveQuestions">
-    You are encouraged to download the proof of submission, which will also contain your latest responses.
+    <b>You are encouraged to download the proof of submission, which will also contain your latest responses.</b>
```

Comment:
I noticed the position of the bold tags has changed, can we shift them to cover only `encouraged ... submission`?